tensorflow_io/grpc/BUILD

```diff
@@ -0,0 +1,65 @@
+licenses(["notice"])  # Apache 2.0
+
+package(default_visibility = ["//visibility:public"])
+
+load("@com_github_grpc_grpc//bazel:cc_grpc_library.bzl", "cc_grpc_library")
+
+genrule(
+    name = "endpoint_py",
+    srcs = [
+        "endpoint.proto",
+    ],
+    outs = [
+        "python/ops/__init__.py",
+        "python/ops/endpoint_pb2.py",
+        "python/ops/endpoint_pb2_grpc.py",
+    ],
+    cmd = "python -m grpc_tools.protoc -Itensorflow_io/grpc --python_out=$(BINDIR)/tensorflow_io/grpc/python/ops/ --grpc_python_out=$(BINDIR)/tensorflow_io/grpc/python/ops/ $< ; touch $(BINDIR)/tensorflow_io/grpc/python/ops/__init__.py",
+    output_to_bindir = True,
```

Comment:
The flatbuffers build for Arrow had an option to put the generated files under a prefix directory, e.g. `out_prefix = "cpp/src/arrow/ipc/"`, is that possible to do here to just put the files in the right place initially?

Reply:
@BryanCutler Bazel used mount point to hide the source directory purposely, so that the generated code could not be exposed in source directory. So we have to copy the python file exposed in  `bazel-bin` (not the source directory). The flatbuffers case is different as the generated file are used for compile (bazel could see), not for final exposure (like python case we encounter here).

Reply:
Ok, I see thanks for the explaination